src/java.base/share/classes/sun/security/util/DerOutputStream.java

```diff
@@ -175,7 +175,7 @@ public void putInteger(BigInteger i) throws IOException {
     /**
      * Marshals a DER integer on the output stream.
      *
-     * @param i the integer in bytes, equivalent to BigInteger::toByteArray.
+     * @param buf buffered data, which must be DER-encoded
```

Comment:
The information of the parameter, "the integer in bytes, equivalent to BigInteger::toByteArray", may be not necessary to be updated, which is right I think.